package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "carpentry",
-  "version": "0.12.2",
```

Comment:
Hey Paul, can you undo this version bump and I'll redo it with npm? Then I can get this fix onto the npm registry for you. BTW I might be back in work next week :)